core/schema.sql

```diff
@@ -2,11 +2,12 @@
 -- PostgreSQL database dump
 --
 
--- Dumped from database version 9.5.0
--- Dumped by pg_dump version 9.5.0
+-- Dumped from database version 9.6.0
+-- Dumped by pg_dump version 9.6.0
 
 SET statement_timeout = 0;
 SET lock_timeout = 0;
+SET idle_in_transaction_session_timeout = 0;
```

Comment:
I think this statement will make pg 9.5 unhappy. idle_in_transaction_session_timeout was added in 9.6.

https://www.postgresql.org/docs/9.6/static/release-9-6.html

So you should prob just delete this line by hand. (sorry)